docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -688,6 +688,26 @@ For details on the expected behaviour when used with the sequence cache clause,
 
 Default: `100`
 
+##### --ysql_yb_fetch_size_limit
+
+Specifies the maximum size (in MB) of total data returned in one response when the query layer fetches rows of a table from DocDB. Used to bound how many rows can be returned in one request. Set to 0 to have no size limit. To enable size based limit, `--ysql_yb_fetch_row_limit` should be set to 0.
+
+If both `--ysql_yb_fetch_row_limit` and `--ysql_yb_fetch_size_limit` are set, then limit is taken as the lower bound of the two values.
```

Comment:
Maybe `If both ... are set,` should be changed to `If both ... are greater than zero`.